runtimes/protocol/lsp.ts

```diff
@@ -1,14 +1,7 @@
-import { ParameterStructures, ProgressType, RegistrationType, RequestType } from 'vscode-languageserver-protocol'
+import { ParameterStructures, ProgressType, RegistrationType, RequestType } from '@aws/language-server-runtimes-types'
 import { _EM } from 'vscode-jsonrpc'
 
-export { TextDocument, Position } from 'vscode-languageserver-textdocument'
-
-// LSP protocol is a core dependency for LSP feature provided by runtimes.
-// Since we aim to provide whole range of LSP specification for Clients and Capabilities,
-// we re-exporting whole LSP protocol for usage.
-// Scoping it down is not practical due to large surface of protocol and types relationship.
-// It will be limiting implementors, if they choose to type their code with more specific types from LSP.
-export * from 'vscode-languageserver-protocol'
```

Comment:
I don't think this change in correct: this effectively moves whole LSP protocol to `types` package, which makes this package meaningless.